TwitchPlaysAssembly/Src/Commands/GameCommands.cs

```diff
@@ -754,9 +754,9 @@ public static void SolveBomb()
 
 		var modules = TwitchGame.Instance.Modules
 			.Where(x => GameRoom.Instance.IsCurrentBomb(x.BombID))
-			.OrderByDescending(module => module.Solver.ModInfo.moduleID.EqualsAny("cookieJars", "organizationModule", "encryptionBingo", "forgetMeLater", "encryptedHangman", "SecurityCouncil"));
+			.OrderByDescending(module => module.Solver.ModInfo.moduleID.EqualsAny("cookieJars", "organizationModule", "forgetMeLater", "encryptedHangman", "SecurityCouncil", "GSAccessCodes"));
 		foreach (var module in modules)
-			if (!module.Solved)
+			if (module.BombComponent != null && !module.Solved)
```

Comment:
A module that doesn't exist anymore shouldn't be in `TwitchGame.Instance.Modules`. Can you revert this line? I can fix Factory so it doesn't cause this.